packages/bpk-component-inset-banner/src/BpkInsetBanner.tsx

```diff
@@ -86,13 +97,70 @@ const BpkInsetBanner = ({
             <BpkText textStyle={TEXT_STYLES.caption}>{subheadline}</BpkText>
           </div>
         </div>
-        <div className={getClassName('bpk-inset-banner--cta-container')}>
-          {callToAction?.text && (
+        {callToAction && callToAction.popoverMessage && (
+          <div
+            role="presentation"
+            className={getClassName('bpk-inset-banner--cta-container')}
+            onClick={(e) => {
+              // Do not propagate the click on the trigger OR popover content up the DOM tree.
+              e.stopPropagation();
+              e.preventDefault();
+            }}
+          >
+            <BpkPopover
+              id={callToAction?.popoverId || ''}
+              label={callToAction?.popverLabel || ''}
+              placement={callToAction?.popoverPlacement || 'bottom'}
+              onClose={(e: {
+                stopPropagation: () => void;
+                preventDefault: () => void;
+              }) => {
+                e.stopPropagation();
+                e.preventDefault();
+              }}
+              aria-label={callToAction?.buttonA11yLabel}
```

Comment:
Will need to check this providing this alongside the label, as looking at the BpkPopover this might compete with `label` which also sets an A11y label for the popover based on the prop

Reply:
Just testing this and this property is having no effect to a screenreader as the `aria-labelledby` inside the popover is being prioritsed over it and using the `label` (which is the popover title) to be the aria for the user :) so this can be omitted

Reply:
Ok I'll remove it :)